src/datachain/lib/dc.py

```diff
@@ -342,7 +342,7 @@ def from_json(
         spec: Optional[DataType] = None,
         schema_from: Optional[str] = "auto",
         jmespath: Optional[str] = None,
-        object_name: str = "",
+        object_name: Optional[str] = "",
```

Comment:
Please not make it optional. An empty value is enough to identify the missing value.

Reply:
FYI `Optional` does not mean the argument is optional. It just means it takes either of. `None` value or some other value.